js/views/RegistrationPages.js

```diff
@@ -123,7 +123,8 @@ const RegistrationPages = React.createClass ({
    */
   beforePageChange(currentPage, nextPage) {
     const shouldContinue = this.validatePage();
-    if (shouldContinue || nextPage < currentPage) {
+    if (shouldContinue || nextPage >= 0 && nextPage < currentPage) {
```

Comment:
I was able to navigate forward, prior to the change, therefore it must be possible for nextPage < currentPage to be true when moving forward.  The only change in this PR is adding the check for nextPage >= 0.  Please explain how this will fix the issue.